pom.xml

```diff
@@ -17,6 +17,16 @@
               <mainClass>com.github.cypher.Main</mainClass>
           </configuration>
       </plugin>
+
```

Comment:
What is the purpose of the changes in this file? Packaging/compiling with _mvn package_ still requires you to specify the main class.